morpheus/utils/nvt/schema_converters.py

```diff
@@ -537,8 +565,37 @@ def _coalesce_ops(graph: nx.Graph,
     return coalesced_workflow
 
 
-def dataframe_input_schema_to_nvt_workflow(input_schema: DataFrameInputSchema,
-                                           visualize: typing.Optional[bool] = False) -> nvt.Workflow:
+def _json_flatten(df_input: typing.Union[pd.DataFrame, cudf.DataFrame], json_cols, preserve_re=None):
+    convert_to_cudf = False
+    if (isinstance(df_input, cudf.DataFrame)):
+        convert_to_cudf = True
+        df_input = df_input.to_pandas()
+
+    json_normalized = []
+    cols_to_keep = list(df_input.columns)
+    for col in json_cols:
+        pd_series = df_input[col]
+        pd_series = pd_series.apply(lambda x: x if isinstance(x, dict) else json.loads(x))
+
+        pdf_norm = pd.json_normalize(pd_series)
+        pdf_norm.rename(columns=lambda x, col=col: col + "." + x, inplace=True)
+        pdf_norm.reset_index(drop=True, inplace=True)
+
+        json_normalized.append(pdf_norm)
+        if (preserve_re is not None and not preserve_re.match(col)):
```

Comment:
```suggestion
        if (preserve_re is None or not preserve_re.match(col)):
```
Need to remove the JSON column from the output DF